tests/jvm/src/test/resources/metacp.expect

```diff
@@ -503,6 +503,16 @@ com.javacp.Test#typeParams(ArrayList,Hashtable).(b) => param b: Hashtable[String
   Hashtable => java.util.Hashtable#
   String => java.lang.String#
   B => com.javacp.Test#[B]
+com.javacp.Test#vararg(Int,Array). => private[javacp] method vararg: (a: Int, args: String*): Unit
```

Comment:
Oops, wait! Why does the symbol say `(Int,Array)` instead of `(Int,String*)`?

Reply:
Ah, good question, the original issue does the same so I followed that example, but I'm still getting acquainted to the concepts so I wasn't sure

Reply:
I've updated the ticket with the expected output. According to the spec (https://github.com/scalameta/scalameta/blob/master/semanticdb/semanticdb3/semanticdb3.md#scala-symbol, see the "Type descriptor is" part), repeated types in parameters should be reflected in the corresponding symbol, too.

Reply:
Yup the expected type descriptor in my original ticket was wrong 😅

Reply:
Makes sense, I'll try to fix this too!